### PR TITLE
Possibility to edit icon size for card brand icon + change R/L side for icon

### PR DESCRIPTION
**Pull request by miraving**
## Change card icon side [68490](https://app.clubhouse.io/vgs/story/68490/ios-sdk-vgscardtextfield-add-possibility-to-edit-card-image-size-and-paddings) 
(**IMPORTANT: Please do not create a Pull Request without creating a story in ClubHouse first**)

## Description of changes in release / Impact of release:
Fucntionality to edit card icon size and change side.

## Screenshots / gif (if customer facing):
(insert image here)

## Value being added to the product or risk being removed by this release
(insert text here)

## Changes to documentation (put link to pull request if some EXISTING docs page needs changes) 
- [ ] My change requires a change to the documentation.
(insert link here)
- [ ] I have updated the documentation accordingly.
(insert link here)

## Backout procedure

Deploy the previous release.

### CODE REVIEW

**Comment by miraving** [comment]
@dmytrokhl I guess the file `docs/_config.yml` should be removed?

File: Sources/VGSCollectSDK/UIElements/Text Field/VGSTextField.swift [hunk 1]
```diff
@@ -149,6 +149,8 @@ internal extension VGSTextField {
         // set main style for view
         mainStyle()
         // text view
+        textField.adjustsFontSizeToFitWidth = true
+        textField.minimumFontSize = 9.0
```
Comment (dmytrokhl) [inline Sources/VGSCollectSDK/UIElements/Text Field/VGSTextField.swift:153]: We will make this attributes public in scope of other PR. Don't set default value for now.

File: framework/Sources/VGSFramework/Extension/UIImage+resize.swift [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+//  VGSFramework
+//
+//  Created by Vitalii Obertynskyi on 20.04.2020.
+//  Copyright © 2020 VGS. All rights reserved.
+//
+
+import Foundation
+
+// unused extension and excluded from target
+internal extension UIImage {
```
Comment (dmytrokhl) [inline framework/Sources/VGSFramework/Extension/UIImage+resize.swift:12]: I assume we don't need this functionality anymore.

File: Sources/VGSCollectSDK/UIElements/Text Field/VGSCardTextField.swift [hunk 3]
```diff
@@ -12,12 +12,72 @@ import UIKit
...
+        case left(size: CGSize)
+        case right(size: CGSize)
+    }
+    
+    internal var originalLeftPadding: CGFloat = -1
+    internal var originalRightPadding: CGFloat = -1
+    
     /// card brand icon width
-    var iconWidth: CGFloat = 45
+    internal var iconSize: CGSize = .zero
```
Comment (dmytrokhl) [inline Sources/VGSCollectSDK/UIElements/Text Field/VGSCardTextField.swift:24]: let's keep the default size as in previous version

File: Sources/VGSCollectSDK/UIElements/Text Field/VGSCardTextField.swift [hunk 4]
```diff
@@ -12,12 +12,72 @@ import UIKit
 
 /// An object that displays an editable text area. Can be use instead of a `VGSTextField` when need to detect and show credit card brand images.
 public class VGSCardTextField: VGSTextField {
+    public enum SideCardIcon {
+        case left(size: CGSize)
```
Comment (dmytrokhl) [inline Sources/VGSCollectSDK/UIElements/Text Field/VGSCardTextField.swift:16]: I should also add left/rigth cases without setting the size. So we can have 4 cases: [left, right, leftWithSize, rightWithSize]

File: Sources/VGSCollectSDK/UIElements/Text Field/Mask/MaskedTextField+padding.swift [hunk 5]
```diff
@@ -24,3 +24,27 @@ extension MaskedTextField {
...
+
+         CGRect textRect = [super rightViewRectForBounds:bounds];
+         textRect.origin.x -= 10;
+         return textRect;
+     }
+     */
+    
+    override func rightViewRect(forBounds bounds: CGRect) -> CGRect {
+        var rect = super.rightViewRect(forBounds: bounds)
+        rect.origin.x -= 5
```
Comment (dmytrokhl) [inline Sources/VGSCollectSDK/UIElements/Text Field/Mask/MaskedTextField+padding.swift:41]: Let's set standard width as default iOS paddings: 8pt.

**Pull request merged**